packages/react/src/runtimes/local/LocalThreadRuntime.tsx

```diff
@@ -213,13 +213,11 @@ export class LocalThreadRuntime implements ThreadRuntime {
     if (message.role !== "assistant")
       throw new Error("Tried to add tool result to non-assistant message");
 
-    let added = false;
```

Comment:
Why is this added check removed?

It checks for repeat calls to addToolResult() to ensure that a roundtrip is only triggered if a tool result is added (and not updated)